armory/art_experimental/attacks/pgd_patch.py

```diff
@@ -3,13 +3,21 @@
 
 
 class PGDPatch(ProjectedGradientDescent):
+    """
+    Apply Masked PGD to image and video inputs,
+    where images are assumed to have shape (NHWC)
+    and video are assumed to have shape (NFHWC)
+    """
+
     def __init__(self, estimator, **kwargs):
         super().__init__(estimator=estimator, **kwargs)
         assert (
             not self.estimator.channels_first
         ), "This attack currently only supports channels last format"
```

Comment:
Only tangentially related but while we're here: I believe we discussed this assertion and determined that it's unnecessary, since inputs are assumed to be in channels-last format (regardless of the estimator being used). Do you agree/am I remembering correctly? If so, we should remove the assertion

Reply:
Yes, that's correct. Please remove the assertion.